lib/scryptenc/scryptenc.h

```diff
@@ -66,6 +66,11 @@ struct scryptenc_params {
 	size_t maxmem;
 	double maxmemfrac;
 	double maxtime;
+
+	/* Precise parameters. */
```

Comment:
"Precise" seems like the wrong word here.  "Explicit"?  "Manually specified"?

Reply:
I've used all 3 at various points in history of this feature.  I'll go with "explicit", then.